build-system/git.js

```diff
@@ -94,7 +94,9 @@ exports.gitBranchName = function() {
  * @return {string}
  */
 exports.gitCommitHash = function() {
-  return getStdout('git rev-parse --verify HEAD').trim();
+  return process.env.TRAVIS_PULL_REQUEST_SHA ?
+    process.env.TRAVIS_PULL_REQUEST_SHA :
```

Comment:
This change LGTM. I suggest adding a comment indicating why we do this (Travis PRs use a merge commit, whose SHA is different from `TRAVIS_PULL_REQUEST_SHA`). Non-blocking though!